lib/AtteanX/Serializer/SPARQLHTML.pm

```diff
@@ -141,6 +141,22 @@ Serializes the L<Attean::API::Term> object as HTML.
 				s/</&lt;/g;
 			}
 			my $html	= $uri;
+
+			if (my $map = $self->namespaces) {
+				my $abr = $map->abbreviate($html);
+
+				if ($abr) {
+					return qq[<a href="${html}">$abr</a>];
+				} else {
+					return $html;
+				}
+
+			} else {
+			
+				return $html;
```

Comment:
I'd like it done whether or not there's a namespace map.

I was planning on merging your test file with t/serializer-sparqlhtml.t anyway. So if you don't want to do that, I can take care of it during the merge.